### DISCUSSION TITLE
Raspberry Pi 4 External Discs are goes to sleep mode.

### DISCUSSION THREAD

**Discussion by ogunkakisim** [Chia-Network/chia-blockchain#8118] [Support]
Hello friends, can anyone help you farm on Raspberry Pi 4 ? My external drives goes to sleep mode when I go to bed at night and wake up in the morningi see external drives are sleeping so its not farming... I tried both Raspberry OS and Ubuntu but I couldn't find a solution. We can turn it off in Windows power settings, but unfortunately I couldn't find it in Linux. If anyone knows the code that can help, can you write it, thanks.

**Comment by BrandtH22** [comment]
@ogunkakisim , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - many linux users have created scripts to perform a quick read of the plot drives to ensure they do not sleep while farming. Let us know if you have any questions regarding this process either by reopening this ticket or on discord.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
